### PR TITLE
Added type hints for the public functions

### PR DESCRIPTION
**Pull request by regebro**

### CODE REVIEW

File: tzlocal/win32.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def get_localzone_name():
     return _cache_tz_name
 
 
-def get_localzone():
+def get_localzone() -> tzinfo:
```
Comment (agronholm) [inline tzlocal/win32.py:124]: This always returns a ZoneInfo, right?
```suggestion
def get_localzone() -> zoneinfo.ZoneInfo:
```

File: tzlocal/win32.py [hunk 2]
```diff
@@ -137,7 +137,7 @@ def get_localzone():
     return _cache_tz
 
 
-def reload_localzone():
+def reload_localzone() -> None:
```
Comment (agronholm) [inline tzlocal/win32.py:140]: This also returns a ZoneInfo, doesn't it?
```suggestion
def reload_localzone() -> zoneinfo.ZoneInfo:
```

**Review by agronholm — CHANGES_REQUESTED** [review]
Some return types should probably be changed.

File: tzlocal/win32.py (hunk 1 above)
Comment (regebro) [reply tzlocal/win32.py:124]: Yeah, that's true, it will always be a ZoneInfo.

**Comment by agronholm** [comment]
Awesome! What's now left is to mark the project as typed by adding the `py.typed` marked (plus the `Typing :: Typed` classifier, although that makes no practical difference).

**Pull request merged**